openfisca_us/variables/finance/benefit/tanf.py

```diff
@@ -38,6 +38,12 @@ class tanf_max_amount(Variable):
     documentation = "The maximum benefit amount a family could receive from Temporary Assistance for Needy Families given their state and family size."
     unit = "currency-USD"
 
+    def formula(spm_unit, period, parameters):
+        family_size = spm_unit("family_size", period)
+        state = spm_unit("state", period)
```

Comment:
```suggestion
        state = spm_unit.household("state_code", period)
```

Though @MaxGhenis should we rename this variable to simply `state`?

Reply:
We can discuss in https://github.com/PolicyEngine/openfisca-us/discussions/326